lib/ui/base/components/progress_bar.dart

```diff
@@ -1,24 +1,18 @@
-import 'dart:async';
-
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
 import 'package:plante/base/base.dart';
 import 'package:plante/base/pair.dart';
-import 'package:plante/ui/base/text_styles.dart';
-import 'package:plante/ui/base/ui_utils.dart';
 
-class ProgressBarWithHints extends StatefulWidget {
+class ProgressBar extends StatefulWidget {
```

Comment:
I think it might be confusing if the class ends up being named as `"ProgressBar"` - the name is very similar to something which the standard library might contain.
Maybe it should be named "`IncrementalProgressBar`" or something similar?

Reply:
yes, i thought so too, let me fix that